.github/workflows/ci.yaml

```diff
@@ -200,8 +251,12 @@ jobs:
         key: ccache-sles-${{ github.ref }}
         restore-keys: ccache-sles-
 
+    - name: Docker Login
+      run: |
+        echo $DOCKER_TOKEN | docker login -u $DOCKER_USER --password-stdin
+
     - name: Build migraphx
-      shell: bash -c "docker run -i -v=$GITHUB_WORKSPACE:/data -w /data migraphx-sles bash < {0}"
+      shell: bash -c "docker run -i -v=$GITHUB_WORKSPACE:/data -w /data $DOCKERIMAGE bash < {0}"
```

Comment:
I don't understand where this $DOCKERIMAGE is being set. both sles and ubuntu should have different tags.

Reply:
For the `sles` job it is assigned on line 240.  Each step inherits the environment variables of previous steps